packages/helm-charts/blockscout/templates/blockscout-web.deployment.yaml

```diff
@@ -43,6 +43,7 @@ spec:
           - name: PORT
             value: "4000"
 {{ include "celo.blockscout-env-vars" .  | indent 10 }}
+{{ include "celo.blockscout-env-vars-web" .  | indent 10 }}
```

Comment:
Do you expect a lot more blockscout env vars just for the web pod? Otherwise, can we just move it into this file?

Reply:
I do not expect more, at least for now. We can add directly and if we add more then we can switch again to this approach.